src/main/cpp/mechanisms/noteManager/decoratormods/noteManager.cpp

```diff
@@ -88,6 +88,10 @@ void noteManager::RunCommonTasks()
 	ResetLauncherAngle();
 	ResetElevator();
 
+	Logger::GetLogger()->LogData(LOGGER_LEVEL::PRINT, string("Sensors"), string("Front"), getfrontIntakeSensor()->Get());
+	Logger::GetLogger()->LogData(LOGGER_LEVEL::PRINT, string("Sensors"), string("Back"), getbackIntakeSensor()->Get());
+	Logger::GetLogger()->LogData(LOGGER_LEVEL::PRINT, string("Sensors"), string("Feeder"), getfeederSensor()->Get());
```

Comment:
Not for today, but lets clean up the logging, but if we want to just approve it. No worries, if not